ramls/dto/initJobExecutionsRqDto.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request to initialize JobExecution entities",
+  "additionalProperties": false,
+  "properties": {
+    "uploadDefinitionId": {
```

Comment:
we dont need relations on data-import module
just 
{
files:[
{name: "marc.bib"},
{name: "test.mrc"}
]
}